rts/Sim/Units/UnitDef.h

```diff
@@ -396,6 +396,8 @@ struct UnitDef: public SolidObjectDef
 
 	int maxThisUnit;                                ///< number of units of this type allowed simultaneously in the game
 
+	float selectionRadius;
```

Comment:
I think adjacent to `selectionVolume` was the better place to keep this - similar things together, plus I could see it being useful for non-units (IIRC in springboard you can drag-select features somehow). Also could be renamed to `dragSelectionRadius` since it's specific to that method.